CMakeLists.txt

```diff
@@ -125,15 +125,67 @@ if("sgx" IN_LIST COMPILE_TARGETS)
   )
 
   add_dependencies(ccf ccf.enclave)
-endif()
 
-if("virtual" IN_LIST COMPILE_TARGETS)
+elseif(COMPILE_TARGET STREQUAL "snp")
```

Comment:
I think we should add a comment here to say that the SNP build uses virtual for now (i.e. `.host` linked libraries), but that this is likely to change in the near future as SNP will require a separate toolchain, compile flags, etc.

Reply:
https://github.com/microsoft/CCF/pull/4367/commits/6b89de802bee3a8468af44636caeb4065ff3906b